app/src/main/java/com/univoice/feature/example/xml/NameInputActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.univoice.feature.example.xml
+
+import android.os.Bundle
+import com.univoice.R
+import com.univoice.core_ui.base.BindingActivity
+import com.univoice.databinding.ActivityNameInputBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+
+@AndroidEntryPoint
+class NameInputActivity : BindingActivity<ActivityNameInputBinding>(R.layout.activity_name_input) {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        initView()
+    }
+
+    override fun initView() {
+
+    }
```

Comment:
```suggestion
    override fun initView() {
        binding.etNameInputName.requestFocus()
    }
```
P1: 첫 editText 포커스 주기!!

Reply:
수정했습니당